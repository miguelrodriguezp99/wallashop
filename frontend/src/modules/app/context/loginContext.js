import { createContext, useState } from "react";

// Este es el que tenemos que consumir
export const LogginContext = createContext();

// Este es el que nos provee de acceso al contexto
export function LogginProvider({ children }) {
	const [token, setToken] = useState(null);
	const [image, setImage] = useState(null);
	const [user, setUser] = useState({});

	return (
		<LogginContext.Provider
			value={{
				token,
				setToken,
				image,
				setImage,
				user,
				setUser,
			}}
		>
			{children}
		</LogginContext.Provider>
	);
}
