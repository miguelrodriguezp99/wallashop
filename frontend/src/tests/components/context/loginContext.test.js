import React from 'react';
import { render } from '@testing-library/react';
import { LogginProvider } from '../../../modules/app/context/loginContext';

test('LogginProvider renders correctly', () => {
    const { asFragment } = render(<LogginProvider />);
    expect(asFragment()).toMatchSnapshot();
  });