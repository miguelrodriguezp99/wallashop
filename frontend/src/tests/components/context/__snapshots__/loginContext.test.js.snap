// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogginProvider renders correctly 1`] = `<DocumentFragment />`;
